WeatherService.Tests/WeatherMonitorTesting/BotFactoryTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using WeatherService.Bots;
+using WeatherService.Models;
+
+namespace WeatherService.Tests.WeatherMonitorTesting;
+
+public class BotFactoryTests
+{
+    private readonly Fixture _fixture;
+    private readonly BotConfig _botConfig;
+
+    public BotFactoryTests()
+    {
+        _fixture = new Fixture();
+        _botConfig = _fixture.Create<BotConfig>();
+    }
+
+
+    [Theory]
+    [InlineData("RainBot")]
+    [InlineData("SnowBot")]
+    [InlineData("SunBot")]
+    public void CreateBotList_ShouldReturnListOfBots_WhenBotTypesAreValid(string validBot)
```

Comment:
let's have consistent naming convention for the test methods: MethodName_Input_BehaviorOrOutput
```suggestion
    public void CreateBotList_WhenBotTypesAreValid_ShouldReturnListOfBots(string validBot)
```